README.md

```diff
@@ -87,6 +88,20 @@ above.
 - `atom-notes:interlink`: Jumps to referred note when the cursor is on
   an `[[interlink]]`.
 
+## Triggering from outside Atom
+
+To add Atom Notes to the Apple Services menu use this [Apple service][apple-service].
+
+Otherwise in macOS and Windows the URL `atom://atom-notes/toggle` will toggle the notes view.
+The command will operate in the front-most or most recently active window or open a new one. It will
+start Atom if necessary. There are many ways to automate this. For example, in macOS:
+
+- Call `open atom://atom-notes/toggle` from the command line or a script.
+- Use the [Apple service][apple-service] mentioned above.
+- Install [Alfred][alfred] (requires the Power Pack purchase) and the
+[alfred-atom-notes workflow][alfred-atom-notes]. Then use the configurable
+shortcut, shift-cmd-j, from inside or outside Atom to toggle the notes view.
```

Comment:
Here it might be worthwhile to not even mention `shift-cmd-j` as that shortcut was only an example given in the above section. We wouldn't want to start getting issues from people trying to use the shortcut to trigger `atom-notes` from outside Atom and it not working for them because they never set up a shortcut.

Perhaps something like:

```
Then use your configured shortcut &mdash; see the section on
Keybindings, above, for details on configuration a shortcut &mdash; from
inside or outside Atom to toggle the notes view.
```